src/components/atoms/Loader.tsx

```diff
@@ -1,4 +1,4 @@
-import { ReactElement } from 'react'
+import React, { ReactElement } from 'react'
```

Comment:
issue(blocking): we no longer need to import React here [iirc](https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj)